dev-utils/karma.js

```diff
@@ -89,9 +89,9 @@ const baseConfig = {
     startConnect: false,
     recordVideo: false,
     recordScreenshots: true,
-    tunnelIdentifier,
+    tunnelName,
     options: {
-      // seleniumVersion: '2.48.2',
+      seleniumVersion: '2.48.2',
```

Comment:
@vigneshshanmugam do you know why this line was commented?

Reply:
With older selenium versions, it wasnt possible to launch latest browsers. We can remove this versioning deps.